ara/templates/playbook_file_summary.html

```diff
@@ -0,0 +1,26 @@
+{% extends "layout.html" %}
```

Comment:
I think there's only ever going to be one primary file ? For a given playbook, that file can be at the top (outside the table or something) and then the task/role files below. It'd be kind of silly to add another column that'll really only be "ticked" for the primary file.